biogeochem/FatesAllometryMod.F90

```diff
@@ -1036,6 +1042,35 @@ subroutine bsap_allom(d,ipft,crowndamage,canopy_trim,elongf_stem, sapw_area,bsap
           end if
        end if
 
+    case(2) ! this is a 'sapwood' function specifically for grass PFT that do not produce
+            ! dead woody biomass. So bsap = bagw. Might remove the bsap and bdead for grass
+            ! in the future as there is no need to distinguish the two for grass above- and belowground biomass
+
+       call bagw_allom(d,ipft,crowndamage,elongf_stem,bagw,dbagwdd)
+       call bbgw_allom(d,ipft, elongf_stem,bbgw,dbbgwdd)
+       bsap = bagw + bbgw
+
+       ! replicate the crown damage code
+       ! Do we really need this for grass? I would think this can be helpful for
+       ! grazing in the future. --XLG
+       if(crowndamage > 1)then
```

Comment:
tagging @JessicaNeedham

Reply:
Agree @XiulinGao that we should think ahead about possible grazing functionality. But as written the crown damage module doesn't change plant height, only crown area and the biomass of woody tissues,  so I'm not sure how applicable it is to grasses. Maybe we could skip this for now, but think about making the damage code more general so it could be used in the future?

Reply:
Per discussion in the fates software meeting today @rgknox will make a PR to this branch to remove this and include a graceful failure.

Reply:
I see. In that case then I think this is not that helpful for grass grazing. I agree that we can remove this for grasses for now.